lib/moonshot/stack_template.rb

```diff
@@ -10,30 +8,19 @@ def required?
       end
     end
 
-    attr_reader :body
+    BODY_MAX_LENGTH_IN_BYTES = 51_200
 
     def initialize(filename, log:)
       @log = log
-
-      unless File.exist?(filename)
-        @log.error("Could not find CloudFormation template at #{filename}")
-        raise
-      end
-
-      # The maximum TemplateBody length is 51,200 bytes, so we remove
-      # formatting white space.
-      @body = JSON.parse(File.read(filename)).to_json
+      @filename = Dir[filename].first
     end
 
-    def parameters
-      JSON.parse(@body).fetch('Parameters', {}).map do |k, v|
-        Parameter.new(k, v['Default'])
-      end
+    def validate
```

Comment:
I introduced this function, I don't know if we need it or not (probably out of scope of this PR).